tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -317,7 +320,14 @@ func labelStoreCommandFunc(cmd *cobra.Command, args []string) {
 		return
 	}
 	prefix := fmt.Sprintf(path.Join(storePrefix, "label"), args[0])
-	postJSON(cmd, prefix, map[string]interface{}{args[1]: args[2]})
+	labels := make(map[string]interface{})
+	for iter := 1; iter < len(args); iter += 2 {
```

Comment:
iter -> i

Reply:
OK, actually when I wrote the code I was thinking 'i' maybe simple so I use the 'iter' which has specific meaning.